api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java

```diff
@@ -409,9 +409,9 @@ public void deletePatientFlagsForPatient(Patient patient) throws DAOException {
 		
 		@SuppressWarnings("unchecked")
 		List<PatientFlag> flags = criteria.list();
-		for (PatientFlag patientFlag : flags) {
+		flags.forEach(patientFlag->{

```

Comment:
Format the code like this:
`flags.forEach(patientFlag -> {`

Reply:
I formatted the code by adding spaces